johnsonandjohnson/Bodiless-JS
# fix(vital-templates): Fix Template Names

## Changes
- Fixed template names 
- Default -> Generic
- Add meta title to PDP tempalte

## Test Instructions

- In edit mode, on home page, click page | Template &  confirm you see the correct list of template names
- In edit mode, add a new page, click page | Template and choose each template and confirm you get expected templated.
